src/tls.rs

```diff
@@ -128,15 +128,15 @@ pub struct IdentityStore(Vec<u8>, String);
 impl TlsSettings {
     pub fn from_config(
         config: &Option<TlsConfig>,
-        require_ident: bool,
+        for_server: bool,
```

Comment:
Does this mean that this setting is _for a server_? aka if we are a server we set this to true?

Also looks like we are not updating any other rust files where before this boolean meant something else, just want to make sure that is correct?

Reply:
Correct, setting this to `true` is for servers. I renamed it because it controls both requiring an identity (certificate + key) in the configuration as well as changing the defaults for certificate verification. The only users of `from_config` that would set this to `true` are the server sources, so no changes to them were required.

Reply:
Just for the future, can we add a comment what these params mean since I feel like people stumbling across this in the future might be confused.

Reply:
Absolutely, I was already thinking this as you were asking.